starter-code/src/app/movies.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Movie, movieList } from './movies-data';
+
+@Injectable()
+export class MoviesService {
+  
+  movies: Array<Movie> = [...movieList];
+
+  constructor() { }
+
+  getMovies() {
+    return this.movies;
+  }
+
+  getMovie(id: string) {
+    let movieInfo;
+
+    this.movies.forEach((oneMovie) => {
```

Comment:
You could also use the JS array `find` method:
`this.movies.find(oneMovie => oneMovie.id === id)` which would evaluate to the right movie directly